resources/todomvc/vanilla-examples/javascript-web-components/src/components/todo-item/todo-item.component.js

```diff
@@ -49,6 +52,13 @@ class TodoItem extends HTMLElement {
                     if (this.id !== undefined)
                         this.item.id = `todo-item-${this.id}`;
                     break;
+                case "index":
+                    if (this.index !== undefined) {
+                        this.item.classList.add(`li-${this.index}`);
+                        this.displayTodo.classList.add(`view-${this.index}`);
```

Comment:
Do we use these classnames anymore?

Reply:
Kind of. They are the classes that the generated css would match if it wasn't for the shadow roots.